sensor_msgs/Image distortedImage
---
sensor_msgs/Image rectifiedImage
